hoomd/hpmc/IntegratorHPMCMonoGPU.cu

```diff
@@ -106,9 +110,8 @@ __global__ void hpmc_shift(Scalar4 *d_postype,
     d_image[my_pidx] = image;
     }
 
-//!< One-thread kernel to accept reject
-__global__ void hpmc_accept(const unsigned int *d_ptl_by_update_order,
-                 const unsigned int *d_update_order_by_ptl,
+//!< Kernel to accept/reject
+__global__ void hpmc_accept(const unsigned int *d_update_order_by_ptl,
```

Comment:
How do the changes to `hpmc_accept` change the performance of normal hard particle simulations?